File: src/1816-Truncate-Sentence/truncate_sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
  let words = s.split(" ")
  let ans = words.slice(0, k).join(" ")

  return ans
};

let s = "Hello how are you Contestant"
let k = 4

console.log(sumOfMultiples(n))

File: src/1816-Truncate-Sentence/truncate_sentence.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        words: str = s.split()
        ans: str = ' '.join(words[:k])

        return ans

if __name__ == "__main__":
    s: str = "Hello how are you Contestant"
    k: int = 4
    solution = Solution()

    print(solution.truncateSentence(s, k))